src/components/index/header/header.js

```diff
@@ -18,6 +18,7 @@ const Header = (props) => {
   const { isDarkTheme } = useColorMode();
   return (
     <div className={styles.container}>
+      <h1 style={{ display: 'none' }}>Luos is an embedded microservice orchestrator</h1>
```

Comment:
Why hide this HTML tag ?

Reply:
Its a request from @lorenzi-nicolas for SEO

Reply:
@lorenzi-nicolas Hiding a H1 isn't forbidden ?

Reply:
This is not the best way to do it, we tried to find a way to solve the appearing/disappearing problem.
Thinking about it further, it would be less impactful for SEO to set the fixed and visible value of h1 to Luos and make the "is" dynamic with the rest of the sentence.
To summarize, remove the hidden h1 and change the dynamic sentences that are displayed to include the "is ». Luos will be the only h1 on our landing page.